packages/core/mesh/teleport/src/muxing/muxer.ts

```diff
@@ -416,6 +418,7 @@ export class Muxer {
           id: channel.id,
           tag: channel.tag,
           contentType: channel.contentType,
+          bufferSize: channel.buffer.length,
```

Comment:
perhaps rename to `writeBufferSize` (if it is the write buffer) to avoid any possible confusion